src/frontend/app/store/helpers/autoscaler-helpers.ts

```diff
@@ -0,0 +1,94 @@
+import moment from 'moment-timezone';
+
+export const PolicyDefaultSetting = {
+  breach_duration_secs_default: 120,
+  cool_down_secs_default: 120,
+};
+export const ScaleTypes = ['upper', 'lower'];
+export const UpperOperators = ['>', '>='];
+export const LowerOperators = ['<', '<='];
+
+export function transformArrayToMap(newPolicy, timezone) {
```

Comment:
As all the exported functions in the files are available throughout stratos they should be named a little bit more specific to the autoscaler. For instance, this one could be `autoscalerTransformArrayToMap`